### ISSUE TITLE
Version 3.0.0 documentation

### ISSUE THREAD

**Issue by dcordero** [robb/Cartography#280]
As documented in [Readme.md](https://github.com/robb/Cartography#versioning) the expected Cartography version to use for each version of the swift language are:

```
For Swift 3.x: 1.1.0
For Swift 4.x: 2.0.0
```

I see that 3.0.1 was actually released. 
Is 3.0.0 a dev version not intended to be used in production? or is it just a not updated documentation?

**Comment by orta** [comment]
No, those are only for the older last per-swift versions, the full changes are here: https://github.com/robb/Cartography/releases

**Comment by dcordero** [comment]
I see, it makes sense thanks for your reply @orta 

Would it make sense to rephrase it to make it more clear? 

I mean something like this:
```diff 
## Versioning

-For *Swift 3.x*: 1.1.0
+For *Swift 3.x*: Versions <= 1.1.0

-For *Swift 4.x*: 2.0.0
+For *Swift 4.x*: Versions >= 2.0.0
```

**Comment by orta** [comment]
Sure thing, send a PR?

**Comment by dcordero** [comment]
Sure, and thanks a lot for your time 

PR: #282
